draft-packages/form/KaizenDraft/Form/Primitives/Input/Input.tsx

```diff
@@ -80,9 +80,8 @@ const Input: Input = ({
       disabled={disabled}
       className={classnames(
         styles.input,
+        styles[status],
         {
-          [styles.error]: status === "error",
-          [styles.success]: status === "success",
           [styles.default]: !reversed,
           [styles.reversed]: reversed,
```

Comment:
A suggestion that can be for discussion (not to be implemented here), I think the class would benefit from being prefixed with `theme` since `default` can also be a status (or even other things in the future)

Reply:
Good point, theme could work as a better prop, does depend on how we want to handle reverse, I might hold off in this PR